<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSwiftCore</class>
 <widget class="BlackGui::CSystemTrayWindow" name="CSwiftCore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>swiftcore UI</string>
  </property>
  <widget class="QWidget" name="qw_MainWidget">
   <layout class="QGridLayout" name="gl_Main">
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="gb_Commands">
      <property name="title">
       <string>Commands</string>
      </property>
      <layout class="QVBoxLayout" name="vl_Commands">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="BlackGui::Components::CCommandInput" name="lep_CommandLineInput">
         <property name="placeholderText">
          <string>.dot commands</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QGroupBox" name="gb_Network">
      <property name="title">
       <string>Network</string>
      </property>
      <layout class="QHBoxLayout" name="hl_Network">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="pb_DisconnectNetwork">
         <property name="text">
          <string>disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2" alignment="Qt::AlignBottom">
     <widget class="QGroupBox" name="gb_Controls">
      <property name="title">
       <string>Control</string>
      </property>
      <layout class="QVBoxLayout" name="vl_Controls">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="pb_Restart">
         <property name="styleSheet">
          <string notr="true">/** Main window **/
QTextEdit {
  background-color: DimGray;
}</string>
         </property>
         <property name="text">
          <string>restart</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="BlackGui::Components::CInfoBarStatusComponent" name="comp_InfoBarStatus" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="BlackGui::Components::CCoreInfoAreaComponent" name="comp_InfoArea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="gb_DBus">
      <property name="title">
       <string>DBus Mode (changing requires restart)</string>
      </property>
      <layout class="QVBoxLayout" name="vl_DBus">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="BlackGui::Components::CDBusServerAddressSelector" name="comp_DBusSelector">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="gb_AudioAndCockpit">
      <property name="title">
       <string>Audio/Cockpit</string>
      </property>
      <layout class="QHBoxLayout" name="hl_AudioCockpit">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="pb_AudioAdvanced">
         <property name="text">
          <string>advanced</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Audio">
         <property name="text">
          <string>audio</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mb_SwiftCore">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="menu_SettingsDialog"/>
   </widget>
   <widget class="QMenu" name="menuInternals">
    <property name="title">
     <string>Internals</string>
    </property>
    <addaction name="menu_RawFsdMessageDialog"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
   <addaction name="menu_Settings"/>
   <addaction name="menuInternals"/>
  </widget>
  <action name="menu_SettingsDialog">
   <property name="icon">
    <iconset resource="../misc/misc.qrc">
     <normaloff>:/pastel/icons/pastel/16/wrench-orange.png</normaloff>:/pastel/icons/pastel/16/wrench-orange.png</iconset>
   </property>
   <property name="text">
    <string>Modify settings</string>
   </property>
  </action>
  <action name="menu_RawFsdMessageDialog">
   <property name="text">
    <string>FSD Messages</string>
   </property>
   <property name="toolTip">
    <string>Raw FSD Messages</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CDBusServerAddressSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbusserveraddressselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CCommandInput</class>
   <extends>QLineEdit</extends>
   <header>blackgui/components/commandinput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CInfoBarStatusComponent</class>
   <extends>QWidget</extends>
   <header>blackgui/components/infobarstatuscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CSystemTrayWindow</class>
   <extends>QMainWindow</extends>
   <header>blackgui/systemtraywindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CCoreInfoAreaComponent</class>
   <extends>QWidget</extends>
   <header>blackgui/components/coreinfoareacomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lep_CommandLineInput</tabstop>
  <tabstop>pb_Audio</tabstop>
  <tabstop>pb_AudioAdvanced</tabstop>
  <tabstop>pb_Restart</tabstop>
  <tabstop>pb_DisconnectNetwork</tabstop>
 </tabstops>
 <resources>
  <include location="../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
